### ISSUE TITLE
Incompatibility With A Popular Mod - Any Ideas Why??

### ISSUE THREAD

**Issue by The-Euphoria** [Kashdeya/SimpleUpgrades#5]
Hello, I found your mod a couple of days ago, and it's fantastic! But, it is incompatible with another popular mod that I and a lot of people use, [RTG (Realistic Terrain Generation)](http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/2524489-realistic-terrain-generation-rtg-realistic-biomes).

I spent a few hours last night disabling different things in the SU config to see if I could find what part is incompatible, but I didn't find anything :(

Do you know any reason to why they do not work together, and if so, is there something I could do to fix it???
Thank you for your time, and thank you for this awesome mod :+1:

**Comment by Kashdeya** [comment]
Waiting on crash report.

**Comment by The-Euphoria** [comment]
Kashdeya, I really cannot explain this. I tested RTG and SU 1.0..6 together and they simply wouldn't play nice. So I removed RTG and tested SU 1.0.6 on it's own which worked perfectly. Now since re-adding RTG to my mods list to generate a new crash report, they are playing nice with each other and haven't crashed since. 8 new worlds created, flying around for a good few hours and nothing... all seems fine now :confused:

Really weird that they suddenly started working fine with each other, and it only took removing RTG then adding it back. No idea how that fixed the issue. Sorry for the inconvenience.

**[Edit] **- On a separate note, to save opening a new bug report I'll just use this thread. "Dense Lapis" is also broken with 1.0.6. as well as the "Diamond tipped hammer", as reported by the other users earlier today. The Dense Lapis texture when in-hand and in the inventory is missing, but the texture on the stone if fine.

Thanks again
